pytorch_lightning/core/hooks.py

```diff
@@ -86,17 +84,13 @@ def on_before_zero_grad(self, optimizer):
             model.on_before_zero_grad(optimizer) # < ---- called here
             optimizer.zero_grad
 
-        :param optimizer:
-        :return:
+        :param optimizer: The optimizer optimizer for which grads should be zeroed.
```

Comment:
pls use napoleon style

Reply:
ok, i keep it in mind. i guess someone should do a full pass over all documentation and make it consistent. can open a issue later.

Reply:
sure, that is when it is changing so let's do it correctly...
I also think that having separate PR fro typing and docs are better, keep the work simple and straight forward...